newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['SDC-Jasper-Service'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'c73b804e50b6ee3984dfb686909caf7730ecNRAL',
+  logging: {
```

Comment:
Ah, you posted the license key too! I'm not sure if it's a huge deal or not, but you will want to at least .gitignore and rm --cached

Reply:
thank you Jay!